<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionDialog</class>
 <widget class="QDialog" name="QuestionDialog">
  <property name="windowTitle">
   <string>Nueva pregunta – MCQ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_subject">
       <property name="text">
        <string>Materia:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="w_top">
       <layout class="QHBoxLayout" name="layout_top">
        <property name="leftMargin"><number>0</number></property>
        <property name="topMargin"><number>0</number></property>
        <property name="rightMargin"><number>0</number></property>
        <property name="bottomMargin"><number>0</number></property>
        <item>
         <widget class="QComboBox" name="cb_subject">
          <property name="editable"><bool>true</bool></property>
         </widget>
        </item>
        <item>
         <spacer name="spacer1">
          <property name="orientation"><enum>Qt::Horizontal</enum></property>
          <property name="sizeHint" stdset="0">
           <size><width>20</width><height>20</height></size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_reference">
          <property name="text"><string>Referencia:</string></property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_reference">
          <property name="placeholderText"><string>ej.: exam_1025</string></property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_prompt">
       <property name="text"><string>Enunciado:</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="prompt"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="counter">
       <property name="text"><string>0/3000</string></property>
       <property name="alignment"><set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set></property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_opts">
       <property name="text">
        <string>Opciones (texto, explicación, correcta):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="OptionTable" name="tbl_options">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <layout class="QHBoxLayout" name="layout_add">
       <property name="leftMargin"><number>0</number></property>
       <property name="topMargin"><number>0</number></property>
       <property name="rightMargin"><number>0</number></property>
       <property name="bottomMargin"><number>0</number></property>
       <item>
        <widget class="QToolButton" name="btn_add">
         <property name="text"><string>+</string></property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_del">
         <property name="text"><string>–</string></property>
        </widget>
       </item>
       <item>
        <spacer name="spacer2">
         <property name="orientation"><enum>Qt::Horizontal</enum></property>
         <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Save|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OptionTable</class>
   <extends>QTableWidget</extends>
   <header>examgen/gui/widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
